src/app.js

```diff
@@ -1 +1,8 @@
+/* eslint-disable no-console */
 'use strict';
+
+const minimist = require('minimist');
+
+const { name, lastname, age } = minimist(process.argv.slice(2));
+
+console.log(name, lastname, 'is', age);
```

Comment:
Consider using a template string for more readable output
```suggestion
console.log(`${name} ${lastname} is ${age}`);
```